src/fsharp/xlf/FSStrings.de.xlf

```diff
@@ -1284,27 +1284,27 @@
       </trans-unit>
       <trans-unit id="UnitTypeExpected">
         <source>The result of this expression has type '{0}' and is implicitly ignored. Consider using 'ignore' to discard this value explicitly, e.g. 'expr |&gt; ignore', or 'let' to bind the result to a name, e.g. 'let result = expr'.</source>
-        <target state="needs-review-translation">Das Ergebnis dieses Ausdrucks wird implizit ignoriert. Verwenden Sie ggf. "ignore", um diesen Wert explizit zu verwerfen, z. B. "expr |&gt; ignore" oder "let" zum Binden des Ergebnisses an einen Namen, z. B. "let result = expr".</target>
+        <target state="translated">Das Ergebnis dieses Ausdrucks weist den Typ "{0}" auf und wird implizit ignoriert. Verwenden Sie ggf. "ignore", um diesen Wert explizit zu verwerfen, z. B. "expr |&gt; ignore" oder "let" zum Binden des Ergebnisses an einen Namen, z. B. "let result = expr".</target>
         <note />
       </trans-unit>
       <trans-unit id="UnitTypeExpectedWithEquality">
         <source>The result of this equality expression has type '{0}' and is implicitly discarded. Consider using 'let' to bind the result to a name, e.g. 'let result = expression'.</source>
-        <target state="needs-review-translation">Das Ergebnis dieses Gleichheitsausdrucks wird implizit verworfen. Verwenden Sie ggf. "let", um das Ergebnis an einen Namen zu binden. Beispiel: "let Ergebnis = Ausdruck".</target>
+        <target state="translated">Das Ergebnis dieses Gleichheitsausdrucks weist den Typ "{0}" auf und wird implizit verworfen. Verwenden Sie ggf. "let", um das Ergebnis an einen Namen zu binden. Beispiel: "let Ergebnis = Ausdruck".</target>
         <note />
       </trans-unit>
       <trans-unit id="UnitTypeExpectedWithPossiblePropertySetter">
         <source>The result of this equality expression has type '{0}' and is implicitly discarded. Consider using 'let' to bind the result to a name, e.g. 'let result = expression'. If you intended to set a value to a property, then use the '&lt;-' operator e.g. '{1}.{2} &lt;- expression'.</source>
```

Comment:
ok quick q: how can a "equality expression" have any other type than bool? /cc @dsyme @isaacabraham

Reply:
I suppose you could redefine the equality operator

Reply:
Nice one. Also: evil ;-)